android-sdk/README.md

```diff
@@ -0,0 +1,168 @@
+
+# Moloco Android SDK - Android
+Moloco Android SDK provides ad serving for mobile applications to facilitate advertising for app publishers.
```

Comment:
Here and below, I'm not sure but I guess `Moloco Android SDK` should not be a naked noun, even though it looks like a proper noun. If it's possible, we could ask a native speaker to proof-read this.

Reply:
I brought this naming by referencing MoPub Android SDK( https://github.com/mopub/mopub-android-sdk ).
Let me know if you have any other idea.

Reply:
Yes, I think they agree with me.

Reply:
I am not sure I understood your comment properly, so tell me if there are any difference with your idea.
I understood as below.
* Keep title as `Moloco Android SDK`
* Use `Moloco SDK` on the description on here and below.

Reply:
I have asked it to a native speaker, and got a response a naked noun is more natural.
Therefore, I will keep this as origin without modification this time.
Thank you for your comment!